src/client/mod.rs

```diff
@@ -246,6 +121,8 @@ impl TeeConnection {
                 _ocsp_response: &[u8],
                 _now: UnixTime,
             ) -> Result<rustls::client::danger::ServerCertVerified, rustls::Error> {
+                // TODO: DO RA-TLS verification here
```

Comment:
Looks like you implemented the verification? Can the TODO be removed?

Reply:
right :)

Reply:
removed